mtf/metadata/tmet/common.py

```diff
@@ -0,0 +1,291 @@
+from __future__ import print_function
+import yaml
+import os
+import glob
+import sys
+from mtf.common import is_debug
+from avocado.utils import process
+from urlparse import urlparse
+
+MFILENAME = "metadata.yaml"
+DESC = "description"
+SOURCE = "source"
+TESTE = "tests"
+TESTC = "tests_coverage"
+DOCUMENT = ["document", "metadata"]
+SUBTYPE = "subtype"
+SUBTYPE_G = "general"
+SUBTYPE_T = "test"
+BACKEND = "backend"
+TAGS = "tags"
+RELEVANCY = "relevancy"
+DEPENDENCIES = "deps"
+COVPATH = "coverage_path"
+MODULELINT = "enable_lint"
+IMPORT_TESTS = "import_tests"
+TAG_FILETERS = "tag_filters"
+
+
+def print_debug(*args):
```

Comment:
I know. it is connected with pskenar change. because print_debug also translate strings it causes errors when general usage, psklenar does some improvements for this